core/src/test/java/org/apache/iceberg/TestSplitPlanning.java

```diff
@@ -234,6 +234,22 @@ public void testSplitPlanningWithOffsetsUnableToSplit() {
         "We should still only get 2 tasks per file", 32, Iterables.size(scan.planTasks()));
   }
 
+  @Test
+  public void testBasicSplitPlanningDeleteFiles() {
```

Comment:
Do we store split offsets of Delete files? If so should we be checking the splitting on those boundaries?

Reply:
I think not but could be wrong, I dont see it being set in DeleteFile builder:  https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/FileMetadata.java#L38

Reply:
Hum, I guess we have decided they will be smaller than data files so I guess it doesn't matter.

Reply:
I don't see a reason why we wouldn't store that. I think it was overlooked.
@szehon-ho, can we do that in a follow-up PR? Not urgent, at least we should create an issue.

Reply:
Made: https://github.com/apache/iceberg/issues/6659